packages/astro/src/compiler/codegen/index.ts

```diff
@@ -118,9 +112,12 @@ async function getAttributes(attrs: Attribute[], state: CodegenState, compileOpt
         }
         continue;
       }
-      case 'Text':
-        result[attr.name] = JSON.stringify(getTextFromAttribute(val));
+      case 'Text': {
+        let text = getTextFromAttribute(val);
+        warnIfRelativeStringLiteral(nodeName, attr, text);
```

Comment:
It's worth calling out—if we're warning for every relative string literal, does the user have a way to say "Don't warn me, I know what I'm doing?"

Reply:
If not... why wouldn't we just automatically run `Astro.resolve`?

Reply:
Great points! First, this should definitely be using the logger so users log level choice gets honored.

Also, I should probably soften the language a little so it doesn't come across like an error. I definitely feel the annoyance of getting a warning repeatedly for something that you did on purpose. Any suggestions here? Would a link to docs be less annoying?

Reply:
Ok, I've updated this to use the logger so you can turn it off the same way you can turn off other warnings. I don't think we have a flag to set a specific log level so we should add that in another issue. A way to turn off specific warnings sounds like a nice add as well but I can't think of a great solution to that at the moment (maybe a config file or something)